### ISSUE TITLE
Ryzen 3xxx "High performance power profile" support

### ISSUE THREAD

**Issue by Saroumane** [konkor/cpufreq#135]
Hello,

I just tested cpufreq with a new AMD RYZEN 5 3600X had 6-Core (12 threads) 3.8 GHz (4.4 GHz Max Boost) 
Good point : all 12 "cores" are detected !
But I'm not sure to understand how the "power profile selection" works :

=> when I select "High performance" power profile, the 12 governors are set to "ondemand"  
(confirmed with `$ watch -n 1 "cat /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor"` )

They should be set on "performance governor" right ? (According to this benchmark, it's the best governor for performance : https://www.phoronix.com/scan.php?page=article&item=amd-ryzen-cpufreq&num=3 )

**Comment by konkor** [comment]
@Saroumane Performance governor is fixed frequency for acpi-driver. So Ondemand governor is an analogy for intel_pstate's Performance governor. If you want such profile with acpi's Performance governor you can create it in 1 click. Just set all in GUI and click on `New Profile...`, give it name and click Enter.

**Comment by Saroumane** [comment]
Thanks for the suggestion.
For the record, I created a "Full Performance" profile, in which all governor are set to "performance" and "Frequency Boost" is activated. 
Regarding the CPU cores : I have now frequencies moving between 3.5 Ghz to 4.4 GHz fr each core.
Regarding "real life" effect : this profile has actually eliminated the framerate drops I had below 60 fps in demanding games.
When I'm not gaming, the "Balanced profile" is perfect (no Frequency Boost + Ondemand governors)
And to switch automatically (when game is started) from "ondemand" to "performance" I have discovered "gamemode" application. (Unfortunately it does not activate "Frequency boost")
